### PR TITLE
feat: Support for indented blocks

### PR DESCRIPTION
**Pull request by zharinov**

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -49,6 +16,7 @@ jobs:
         uses: actions/setup-node@v1
```
Comment (HonkingGoose) [inline .github/workflows/pr.yml]:
```suggestion
        uses: actions/setup-node@v2.4.0
```

`actions/setup-node` is stable at `@v2.4.0` now: https://github.com/actions/setup-node/releases

We don't get any GitHub Action updates from Renovate and they are not pinned like on our main repo. So we'll need to check why that is...

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -49,6 +16,7 @@ jobs:
...
         with:
           node-version: ${{ matrix.node-version }}
```
Comment (HonkingGoose) [inline .github/workflows/pr.yml:18]:
```suggestion
          node-version: ${{ matrix.node-version }}
          cache: 'yarn'
```

If you want to use the new cache feature that comes with `setup-node@v2` and higher, use this suggestion.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (HonkingGoose) [reply .github/workflows/pr.yml]: Is Renovate even enabled for this repository?

I see we have a Renovate config file in the repository here: https://github.com/renovatebot/parser-utils/blob/main/.github/renovate.json but that config looks nothing like the one we use on the main `renovatebot/renovate` repository.

Comment (zharinov) [reply .github/workflows/pr.yml]: Hopefully, the cache can be reused for `trunk` too, I'll push one more change for this

**Pull request merged**